### DISCUSSION TITLE
Dev dependencies - eslint and prettier

### DISCUSSION THREAD

**Discussion by Chris-isCoding** [mikhael28/paretOS#97] [General]
Howdy!

I have seen Prettier mentioned few times when others were trying to make changes and I myself had to put ```.prettierrc.json``` with adjusted settings when making code changes.

For example I personally like to use single quotes but it is always best to use consistent formatting style.

The problem might arise as currently we don't have dev dependencies for ```ESlint``` or ```Prettier```. Both of these tools should be installed on project basis and not use global installation which might differ between individuals.

In order to make sure everyone follows the same set of rules and guidelines we might want to consider adding these great tools as dev dependencies.

By including ```.prettierrc.json``` and ```.eslintrc.json``` we could ensure the same rules are followed.

Amazing work by Michael and I really appreciate him sharing the project and his vast knowledge with the world.

Thank you!

**Comment by mikhael28** [comment]
Haha yes, you are the second person to ask me to make a prettier file-it is on my to-do list. Look for that in the next 48 hours. I appreciate the kind words!

**Comment by mikhael28** [comment]
Hi @Chris-isCoding I am closing this issue, and combining it's guidance with the issue regarding TypeScript support. Both of those tickets involves changes to `estlintrc` and `prettier.js`, so it makes sense to combine those two steps into one.
